runtime/solo/src/fee_handler.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{
+	AccountId, AffiliateMaxLevel, AssetId, Assets, Balance, Balances, CasinoJamAffiliates,
+	CasinoJamTournament,
+};
+use ajuna_payment_handler::{
+	AllowAllAssets, AssetGameFeeHandler, VoucherHandler, WithdrawCreditOrVoucher,
+	WithdrawFungibles, WithdrawWhitelistedCredit,
+};
+use frame_support::traits::fungible::{NativeFromLeft, NativeOrWithId, UnionOf};
+use sp_runtime::DispatchError;
+
+pub type NativeAndAssets = UnionOf<
+	Balances,
+	Assets,
+	NativeFromLeft,
+	NativeOrWithId<example_transition::asset::AssetId>,
+	AccountId,
+>;
+
+/// Fee handler facilitating payment in the native currency and with whitelisted assets.
+pub type AjunaAssetFeeHandler<Affiliates, AffiliateMaxLevel, Tournament> = AssetGameFeeHandler<
+	AccountId,
+	NativeAndAssets,
+	WithdrawCreditOrVoucher<
+		WithdrawWhitelistedCredit<
+			AllowAllAssets<NativeOrWithId<AssetId>>,
+			WithdrawFungibles<AccountId, NativeAndAssets>,
+		>,
+		DummyVoucherHandler,
+	>,
+	Affiliates,
+	AffiliateMaxLevel,
+	Tournament,
+>;
+
+pub type CasinoJamFeeHandler =
+	AjunaAssetFeeHandler<CasinoJamAffiliates, AffiliateMaxLevel, CasinoJamTournament>;
+
+pub struct DummyVoucherHandler;
+
+impl VoucherHandler for DummyVoucherHandler {
+	type AccountId = AccountId;
+	type Balance = Balance;
+
+	fn consume_vouchers_from(
+		_account: &Self::AccountId,
+		_amount: Self::Balance,
+	) -> Result<(), DispatchError> {
+		Ok(())
+	}
+}
```

Comment:
Hmm, if we don't want to have vouchers, don't we just want to use the other withdraw type that does not use vouchers?

well, I guess it is a test node, so we don't really care, and as it cannot be used for the extrinsic fee, it is not a DDOS risk.